docs/script.js

```diff
@@ -44,40 +44,130 @@ ReactDOM.render(<Component />, document.getElementById('react-1'));
 
 let Wrapper = React.createClass({
   getInitialState() {
-    return {interval: 2000};
+    return {
+      interval: 2000,
+      transitionTimeout: {
+        enter: 200,
+        leave: 200,
+      },
+    };
   },
   updateInterval(value) {
-    this.setState({interval: parseInt(value)});
+    this.setState({ interval: parseInt(value) });
+  },
+  updateTransitionTimeout(type, value) {
+    if(type === 'enter') {
```

Comment:
Can you add a space after if? `if (type === 'enter') {`

Reply:
sure!